src/musig.rs

```diff
@@ -658,6 +658,7 @@ impl<T: SigningTranscript+Clone> MuSig<T,CosignStage> {
     {
         let theirs = Scalar::from_canonical_bytes(theirs.0)
             .ok_or(SignatureError::ScalarFormatError) ?;
+        let theirs = Option::<Scalar>::from(theirs).ok_or(SignatureError::ScalarFormatError)?;
```

Comment:
:question: Mainly curious why this changed.

This file doesn't compile and didn't previously. This module is only used with the `std` feature. 
```suggestion
        let theirs = Scalar::from_canonical_bytes(theirs.0);
        let theirs = Option::<Scalar>::from(theirs).ok_or(SignatureError::ScalarFormatError)?;
```

Reply:
The `ok_or` on line 660 is a mistake from a bad merge. I removed it. We need to add this because `from_canonical_bytes` now returns a `CtOption` rather than an `Option`. Done.